trieste/acquisition/multiobjective/analytic.py

```diff
@@ -0,0 +1,161 @@
+from itertools import product
+from math import inf
+from typing import Callable, Union
+
+import tensorflow as tf
+from tensorflow_probability import distributions as tfd
+
+from ...data import Dataset
+from ...models import ProbabilisticModel
+from ...type import TensorType
+from ...utils.pareto import Pareto
+from ..function import AcquisitionFunction
+from .function import HypervolumeAcquisitionBuilder, get_reference_point
+
+
+class Expected_Hypervolume_Improvement(HypervolumeAcquisitionBuilder):
+    """
+    Builder for the :func:`hv_probability_of_improvement` acquisition function
+    refer yang2019efficient
+    """
+
+    def __init__(self, ref_pt_calc_method: Union[str, Callable] = "default"):
+        """
+        :param ref_pt_calc_method the method of calculating the reference point,
+         either default or a callable
+        """
+        self._ref_pt_calc_method = ref_pt_calc_method
+
+    def __repr__(self) -> str:
+        return f"Expected Hypervolume Improvement({self._ref_pt_calc_method!r})"
```

Comment:
```suggestion
        return f"ExpectedHypervolumeImprovement({self._ref_pt_calc_method!r})"
```
the result of `repr(x)` should, as much as possible, look like a call to the class constructor. For example, `eval(repr(ExpectedHypervolumeImprovement()))` would actually create an `ExpectedHypervolumeImprovement`